bundles/org.openhab.binding.ring/src/main/java/org/openhab/binding/ring/handler/DoorbellHandler.java

```diff
@@ -76,7 +79,8 @@ public void initialize() {
         // "Can not access device as username and/or password are invalid");
         if (this.refreshJob == null) {
             Configuration config = getThing().getConfiguration();
-            int refreshInterval = (int) config.get("refreshInterval");
+            int refreshInterval = ConfigParser
+                    .valueAsOrElse(config.get("refreshInterval"), BigDecimal.class, BigDecimal.valueOf(500)).intValue();
```

Comment:
This is also a bug in your code, not related to the other changes.

Reply:
In general: It is better to create a config class and use `getConfiguration().as()`. But that can be done later.

Reply:
thanks and fixed in the other 3x device handlers too. I picked this up this AM as soon as I was unblocked and could test again :)